content/en/docs/concepts/services-networking/endpointslice.md

```diff
@@ -0,0 +1,93 @@
+---
+reviewers:
+- freehan
+title: Endpoint Slice
+feature:
+  title: Endpoint Slices
+  description: >
+    Scalable tracking of network endpoints in a Kubernetes cluster.
+
+content_template: templates/concept
+weight: 10
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
```

Comment:
@robscott , is the Endpoint Slices feature in the feature gates reference,
https://deploy-preview-16018--kubernetes-io-vnext-staging.netlify.com/docs/reference/command-line-tools-reference/feature-gates/

Reply:
Missed that initially, thanks for catching that! Should be added now.